.circleci/config.yml

```diff
@@ -48,11 +48,12 @@ jobs:
           paths:
             - ~/.npm/
       - run:
-          name: Run Build
+          name: Run build
           command: |
             npm run build
+
```

Comment:
Twine appears to pick this up from the alpha/beta/prc within the release name as specified in [PEP 440](https://peps.python.org/pep-0440/#pre-releases). I checked in test pypi and alpha/beta releases are correctly tagged as pre-release.